backend/frontend/templates/frontend/auth/page_login.html

```diff
@@ -1,12 +1,14 @@
 {% extends "frontend/auth/base.html" %}
 {% block auth_heading %}Login{% endblock %}
 {% block auth_content %}
-  <form action="" method="post">
+  <form action="" method="post" class="my-4" novalidate>
```

Comment:
@kingraphaii, this is what I am supposed to do:

```
The main challenge with this is that we are using Django forms functionality to render the forms so you'll need to get the backend to render things in a useful way.

Also please make sure that field errors get rendered correctly.
```
- If I let HTML do its default validation, then there is no point in me addressing the last point, because they are doing the same thing. Or am I missing something?